### DISCUSSION TITLE
How to set a fixed TOC (On this page)

### DISCUSSION THREAD

**Discussion by kugarocks** [thuliteio/doks#1290] [Q&A]
Something like this. Is there any ways to configure it. Thank you.

![on-this-page](https://github.com/user-attachments/assets/4bf812c7-688e-489a-bddc-8e20df316c34)

**Comment by h-enk** [comment]
No (not yet). but I like this one so I've raised an issue: https://github.com/thuliteio/doks/issues/1291 (to be picked up)
